src/components/Mana/Mana.tsx

```diff
@@ -3,14 +3,16 @@ import classNames from 'classnames'
 import { Network } from '@dcl/schemas/dist/dapps/network'
 import { Header, HeaderProps } from '../Header/Header'
 import './Mana.css'
+import { Popup } from '../Popup/Popup'
 
-export type ManaProps = {
+export interface ManaProps {
```

Comment:
I may be missing something. Would you help me understand why an interface could be needed here?

Reply:
I think its better to interface when we are creating object types.

Reply:
IMO you can leave it as a type in this case